### ISSUE TITLE
Todo list for VR ablation study

### ISSUE THREAD

**Issue by skalarproduktraum** [moreApi/microscenery#30]
# VR ablation mock testing

# High priority
- [ ] scaling is broken with Zen @moreapi
- [ ] scaling should give feedback on scale parameters (e.g., show 100%, 200%,…) @moreapi
- [ ] volume is hiding menu - fix by disabling depth testing or forcing ordering using RenderingOrder? @skalarproduktraum
- [ ] Point cloud ablation: eraser still needs to work @moreapi
- [ ] default red on objects looks like shit @skalarproduktraum
- [ ] ablation should be disabled when path is not planned - should this be shown to the user as informational message? @moreapi
- [ ] would like to rotate the volume @moreapi
- [ ] slicing @moreapi
- [ ] highlight menu text for menu selection @skalarproduktraum
- [ ] menus need to cleaned up @moreapi
  - [ ] have both point and path tool available
  - [ ] have multiple ablation tools for multiple path
- [ ] tool menu rework @skalarproduktraum
  - [ ] **tool menu interaction with tool still in hand does not work**
  - [ ] center around tool
  - [ ] close on tool grab
  - [ ] increase radius?
  - [ ] open menu only once
  - [ ] remember eraser status
- [ ] centrally ablation planning and execution @moreapi

# Medium priority
- [ ] lag when loading data - would async texture loading help here? @skalarproduktraum @moreapi
- [ ] volume rendering should look better @skalarproduktraum
  - [ ] validate with actual data vs. demo data @moreapi
- [ ] bounding box should look better @skalarproduktraum
- [ ] letters on the bounding box are too large @skalarproduktraum
- [ ] menu might need better naming @moreapi @skalarproduktraum
- [ ] exceptions when selecting stuff in menu could show the short description of the exception on screen @skalarproduktraum
- [ ] teleportation should use controller angle/balistic trajectory - issue might be moving on different levels @skalarproduktraum
- [ ] make controller touch/collision balls more precise (e.g. use pyramid or cone, have wireframe or otherwise halfway transparent small object as tip) @skalarproduktraum @moreapi
- [ ] introduce imaging and ablation mode @moreapi
  - [ ] change man hand context menu accordingly
  - [ ] have laser light on or something

# Nice to have
- [ ] frame rate is too low, but SteamVR/Quest reprojection seems to cancel that out @skalarproduktraum
- [ ] menu close should be an X @moreapi
- [ ] increase nozzle length @moreapi

**Comment by skalarproduktraum** [comment]
Instead of having an X for close, we should have a 🐦 instead 😂
